gcc/rust/expand/rust-derive-clone.cc

```diff
@@ -28,6 +34,15 @@ DeriveClone::clone_call (std::unique_ptr<Expr> &&to_clone)
   // $crate::core::clone::Clone::clone for the fully qualified path - we don't
   // link with `core` yet so that might be an issue. Use `Clone::clone` for now?
   // TODO: Factor this function inside the DeriveAccumulator
+
+  // Interestingly, later versions of Rust have a `clone_fn` lang item which
+  // corresponds to this. But because we are first targeting 1.49, we cannot use
+  // it yet. Once we target a new, more recent version of the language, we'll
+  // have figured out how to compile and distribute `core`, meaning we'll be
+  // able to directly call `::core::clone::Clone::clone()`
+
+  // Not sure how to call it properly in the meantime...
+
   auto path = std::unique_ptr<Expr> (
     new PathInExpression (builder.path_in_expression ({"Clone", "clone"})));
```

Comment:
In the meantime we could generate something like `<to_clone_type as #[lang = "clone"]>::clone(to_clone)` but this is extra work. It's not super difficult, so I'll mark it as a good-first-pr

Reply:
Note that this is important for handling `core` and closing out the milestone though, so it does need to be done "shortly"

Reply:
https://github.com/Rust-GCC/gccrs/issues/3371